packages/hardhat/contracts/strategy/CrabStrategyV2.sol

```diff
@@ -469,80 +418,16 @@ contract CrabStrategyV2 is StrategyBase, StrategyFlashSwap, ReentrancyGuard, Own
     }
 
     /**
-     * @notice owner can set the auction time, in seconds, that a hedge auction runs for
-     * @param _auctionTime the length of the hedge auction in seconds
-     */
-    function setAuctionTime(uint256 _auctionTime) external onlyOwner {
-        require(_auctionTime > 0, "invalid auction time");
-
-        auctionTime = _auctionTime;
-
-        emit SetAuctionTime(_auctionTime);
-    }
-
-    /**
-     * @notice owner can set the min price multiplier in a percentage scaled by 1e18 (9e17 is 90%)
-     * @dev the min price multiplier is multiplied by the TWAP price to get the intial auction price
-     * @param _minPriceMultiplier the min price multiplier, a percentage, scaled by 1e18
+     * @notice owner can set a threshold, scaled by 1e18 that determines the maximum discount of a clearing sale price to the current uniswap twap price
+     * @param _otcPriceTolerance the OTC price tolerance, in percent, scaled by 1e18
      */
-    function setMinPriceMultiplier(uint256 _minPriceMultiplier) external onlyOwner {
-        require(_minPriceMultiplier < 1e18, "min price multiplier too high");
-
-        minPriceMultiplier = _minPriceMultiplier;
-
-        emit SetMinPriceMultiplier(_minPriceMultiplier);
-    }
-
-    /**
-     * @notice owner can set the max price multiplier in a percentage scaled by 1e18 (11e18 is 110%)
-     * @dev the max price multiplier is multiplied by the TWAP price to get the final auction price
-     * @param _maxPriceMultiplier the max price multiplier, a percentage, scaled by 1e18
-     */
-    function setMaxPriceMultiplier(uint256 _maxPriceMultiplier) external onlyOwner {
-        require(_maxPriceMultiplier > 1e18, "max price multiplier too low");
+    function setOTCPriceTolerance(uint256 _otcPriceTolerance) external onlyOwner {
+        // Tolerance cannot be more than 20%
+        require(_otcPriceTolerance <= 2e17, "price tolerance is too high");
 
-        maxPriceMultiplier = _maxPriceMultiplier;
+        otcPriceTolerance = _otcPriceTolerance;
 
-        emit SetMaxPriceMultiplier(_maxPriceMultiplier);
-    }
-
-    /**
-     * @notice get current auction details
-     * @param _auctionTriggerTime timestamp where auction started
-     * @return if strategy is selling wSqueeth, wSqueeth amount to auction, ETH proceeds, auction price, if auction direction has switched
-     */
-    function getAuctionDetails(uint256 _auctionTriggerTime)
```

Comment:
Should we add a view function that tells the manager how much to hedge, maybe based on % of stratey to hedge or something.

Ie howMuchTohedge(50%) returns (true, 1000) (isSellingOSqth, quantity)

Reply:
frontend can calculate this right? based on the vault collateral and shortAmount. What is the advantage of a view function on the contract?

Reply:
Maybe just for convenience.  It could be helpful for f/e to calculate it as well as let someone else (MM) or manager check the quantities that we are using.